scrapy_azure_exporter/azure_pipeline.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from scrapy.pipelines.files import FilesPipeline as FSPipeline
+
+from scrapy_azure_exporter.azure_store import AzureFilesStore
+
+logger = logging.getLogger(__name__)
+
+
+class FilesPipeline(FSPipeline):
+    @classmethod
+    def from_settings(cls, settings):
+        azure_store = AzureFilesStore
+        azure_store.AZURE_CONNECTION_STRING = settings.get("AZURE_CONNECTION_STRING")
+        azure_store.AZURE_ACCOUNT_URL_WITH_SAS_TOKEN = settings.get(
+            "AZURE_ACCOUNT_URL_WITH_SAS_TOKEN"
+        )
+        azure_store.AZURE_ACCOUNT_URL = settings.get("AZURE_ACCOUNT_URL")
+        azure_store.AZURE_ACCOUNT_KEY = settings.get("AZURE_ACCOUNT_KEY")
+
+        # NOTE: The "http" schema is necessary to compatibility with Azurite
+        cls.STORE_SCHEMES.update({"azure": azure_store, "http": azure_store})
+        return cls(settings.get("FILES_STORE"), settings=settings)
```

Comment:
I think that, rather than defining such a class, we should document how to configure the `AzureFilesStore` storage class with upstream Scrapy code instead. Users should be able to use Azure the same way they use Google Cloud, and the upstream [GCSFilesStore](https://github.com/scrapy/scrapy/blob/ada917307844950a81226f020b596d5932187f6e/scrapy/pipelines/files.py#L197) class does not require a custom files pipeline (I hope).

Reply:
Hello, have you been able to export images to azure? How did you define FilesStore settings?

Reply:
@Gallaecio The idea of a custom FilesPipeline is only to add the "azure://" scheme to the `STORE_SCHEMES` attribute and initialize the `AzureFilesStore`, as it can be verified here: https://github.com/scrapy/scrapy/blob/ada917307844950a81226f020b596d5932187f6e/scrapy/pipelines/files.py#L332. I was trying to attend this request https://github.com/scrapy/scrapy/issues/2066, so in this case we need to use the Pipeline.

Reply:
Instead of doing that through a pipeline, we should consider documenting the steps that users need to do to get that done manually.